26_Day/26_python_web.md

```diff
@@ -79,11 +79,27 @@ pip install virtualenv
 
 Step 2:
 
+For Windows environment:
+
+```sh
+C:\Users\User\Documents>mkdir python_for_web
+C:\Users\User\Documents>cd python_for_web
+C:\Users\User\Documents\python_for_web>python3 -m venv env
+C:\Users\User\Documents\python_for_web>env\Scripts\activate
```

Comment:
[suggestion]
```suggestion
C:\Users\User\Documents$ mkdir python_for_web
C:\Users\User\Documents$ cd python_for_web
C:\Users\User\Documents\python_for_web$ python3 -m venv env
C:\Users\User\Documents\python_for_web$ env\Scripts\activate
```
it would be interesting to maintain the standard 👍🏽 what do you think ?